lite2/client.go

```diff
@@ -561,15 +561,37 @@ func (c *Client) verifyHeader(newHeader *types.SignedHeader, newVals *types.Vali
 			panic(fmt.Sprintf("Unknown verification mode: %b", c.verificationMode))
 		}
 	} else {
-		// 2) Otherwise, perform backwards verification
-		// Find the closest trusted header after newHeader.Height
-		var closestHeader *types.SignedHeader
-		closestHeader, err = c.trustedStore.SignedHeaderAfter(newHeader.Height)
+		var firstHeaderHeight int64
+		firstHeaderHeight, err = c.FirstTrustedHeight()
 		if err != nil {
-			return errors.Wrapf(err, "can't get signed header after height %d", newHeader.Height)
+			return errors.Wrapf(err, "can't get first header height for verification")
+		}
+		var closestHeader *types.SignedHeader
+		if newHeader.Height < firstHeaderHeight {
+			closestHeader, err = c.TrustedHeader(firstHeaderHeight)
```

Comment:
```suggestion
			firstHeader, err = c.TrustedHeader(firstHeaderHeight)
```

Reply:
I just used closest header because it was already declared and it also is technically the closest header to the new header